plugins/uefi/fu-uefi-devpath.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2018-2019 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#pragma once
+
+#include <glib.h>
+
+G_BEGIN_DECLS
+
+typedef enum {
```

Comment:
Add Gtk-Doc stuff here?

Reply:
I don't think it needs it -- there's only really one flag and the gtk-doc scanner isn't looking in plugins/ at all.